client/js/components/NoProgramCode/styles.js

```diff
@@ -0,0 +1,64 @@
+import {StyleSheet, Dimensions} from "react-native";
+import {colors, fonts} from "../../config/styles";
+
+const styles = StyleSheet.create({
+  description: {
+    fontSize: 26,
+    fontWeight: "400",
+    fontFamily: fonts.bold,
+    textAlign: "center"
+  },
+  message: {
+    padding: 10,
+    paddingTop: 25,
+    color: colors.green,
+    fontSize: 16,
+    textTransform: "uppercase",
+    fontFamily: fonts.light,
+    textAlign: "center"
+  },
+  messageBox: {
+    paddingTop: "50%",
```

Comment:
Use flexbox properties to put your messageBox in the centre instead of paddingTop 50%. Please make the changes.